src/CribblyBackend/Services/StandingsService.cs

```diff
@@ -0,0 +1,47 @@
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Linq;
+using CribblyBackend.DataAccess.Models;
+using CribblyBackend.DataAccess.Repositories;
+
+namespace CribblyBackend.Services
+{
+    public interface IStandingsService
+    {
+        Task<Team> Calculate(Team team);
+    }
+
+    public class StandingsService : IStandingsService
+    {
+        private readonly IGameRepository _gameRepository;
+        public StandingsService(IGameRepository gameRepository)
+        {
+            _gameRepository = gameRepository;
+        }
+        public async Task<Team> Calculate(Team team)
+        {
+            var allGames = await _gameRepository.GetByTeamId(team.Id);
+            team.PlayInGames = allGames.Where(g => g.GameRound <= Game.Round.Round3).ToList();
+            team.BracketGames = allGames.Where(g => g.GameRound >= Game.Round.TourneyRound1).ToList();
+
+            team.Wins = allGames.Where(g => g.Winner != null && g.Winner.Name == team.Name).ToList().Count;
+            team.Losses = allGames.Where(g => g.Winner != null && g.Winner.Name != team.Name).ToList().Count;
```

Comment:
I'm not _exactly_ sure on this, but I think the performance might be slightly better if you omit `ToList` and simply call `allGames.Count(g => g.Winner != null && g.Winner.Name == team.Name);`

Also, might it be better to use `team.Id` and `g.Winner.Id` since `Id` is guaranteed to be unique?

Reply:
So the tests keep failing when I use ```team.Id``` and ```g.Winner.Id```. Will need to keep tinkering to get that to work

Reply:
Probably because your test teams both have the same Id 🙂 
https://github.com/cszczepaniak/cribbly-backend/blob/6fd1e55857b2ad6543b3c61ea901ec4cc3adf5bc/CribblyBackend.UnitTests/ServiceTests/StandingsServiceTests.cs#L17-L18

Reply:
Nvm, figured it out - wrote the test wrong!